templates/test/running.html.ep

```diff
@@ -70,50 +72,27 @@
             </tr>
             % }
 	    % if ($job->backend) {
-            <tr>
-              <td colspan="2" style="padding: 0 0 <%= ($job->backend eq 'kvm2usb')?'8px':'0' %> 0;"><%= $job->backend %></td>
-            </tr>
-            % if($job->backend eq 'kvm2usb') {
-                <tr>
-                    <td style="width: 16px;"><img src="/images/hw_icons/slot.svg" width="16" height="16" title="slot" alt="slot"/></td>
-                    <td><%= $backend_info->{'hwslot'} %></td>
-                </tr>
-                <% if(defined $backend_info->{'hw_info'}) { %>
-                    <% my $hw_info = $backend_info->{'hw_info'}; %>
-                    % for my $hw_key ( ('vendor', 'name', 'cpu', 'cpu_cores', 'memory', 'disks') ) {
-                        % next unless (defined $hw_info->{$hw_key});
-                        <tr>
-                            <td><img src="/images/hw_icons/<%= $hw_key %>.svg" title="<%= $hw_key %>" width="16" height="16" alt="<%= $hw_key %>" /></td>
-                            <td><%= $hw_info->{$hw_key} %></td>
-                        </tr>
-                    % }
-                    % if(defined $hw_info->{'comment'}) {
-                        <tr>
-                            <td colspan="2" style="padding: 8px 0 0 0;"><%= $hw_info->{'comment'} %></td>
-                        </tr>
-                    % }
-                % }
-	     % }
+              <tr>
+                <td colspan="2" style="padding: 0 0 <%= ($job->backend eq 'kvm2usb')?'8px':'0' %> 0;"><%= $job->backend %></td>
+              </tr>
+              % if($job->backend eq 'kvm2usb') {
+	        %= include 'running_kvm2usb'
+              % }
 	    % }
         </table>
-	</div>
+    </div>
 </div>
 
 <div class="grid_12 live-content omega">
-                %= include 'layouts/info'
-                % if (my $msg = flash 'code') {
-                  <blockquote class="ui-state-highlight" style="margin-bottom: 0.6em;"><%== $msg %></blockquote>
-                % }
+  %= include 'layouts/info'
```

Comment:
Is this indented following our (insane) standard for templates? It's not clear in github